bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -769,32 +771,35 @@ std::pair<PrePrepareMsg *, bool> ReplicaImp::finishAddingRequestsToPrePrepareMsg
           numOfTransientPreprepareMsgs_++;
         } else {
           LOG_ERROR(GL, "The number of transient preprepare messages are very large so they will have to be retried.");
-          delete prePrepareMsg;
           return std::make_pair(nullptr, false);
         }
+        // prePrepareMsg is about to be moved. save here values for future use
+        auto seqNum = prePrepareMsg->seqNumber();
```

Comment:
nit: I would remove these 4 lines. inflates code.
Whats the problem of having redirection and function call in kv pairs seen in log?
if we have a habit of helper vars before each log entry, we will enflate code x2.
In general, it's a bad idea to do so.
you may say "only here".
but if everyone will do it we will get files of 3000 lines instead of 1500 + code maintainance efforts explode.

Reply:
Hi, I saved these 4 variables since they are taken from the message which is moved before the vars are used (as mentioned in the comment above). What is your suggestion?

Reply:
just ignore this